src/coreclr/jit/gentree.cpp

```diff
@@ -2671,6 +2671,20 @@ bool GenTree::Compare(GenTree* op1, GenTree* op2, bool swapOK)
 
     if (kind & GTK_UNOP)
     {
+        if (oper == GT_IND)
+        {
+            if (op1->TypeIs(TYP_STRUCT))
+            {
+                // Rare case -- need contextual information to check equality in some cases.
+                return false;
```

Comment:
Can you remind me the case with IND<struct>? was that something around GT_RETURN?

Reply:
There is only occurrence now: `STORE_DYN_BLK` source if its a `memcpy`.